go.mod

```diff
@@ -1,8 +1,12 @@
-module github.com/open-cluster-management/policy-generator-plugin
+module open-cluster-management.io/ocm-kustomize-generator-plugins
 
 go 1.16
 
 require (
+	github.com/go-openapi/jsonpointer v0.19.5 // indirect
```

Comment:
Do you know why these indirect dependencies are not showing up when they weren't before?

Reply:
I'm not sure. They appeared when I ran `go mod tidy`, so maybe that hadn't been run before?